domains/harnish.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "HarnishSavsani",
+        "email": "harnishsavsani9071@gmail.com"
+    },
+    "records": {
+        "A": ["216.198.79.1"]
```

```suggestion
        "CNAME": "harnishsavsani.vercel.app"
```
Please link to your website.